@types/modules/stpm/minievents.d.ts

```diff
@@ -0,0 +1,33 @@
+declare interface METMMiniEventUpdateRequest {
+    eventName?: string;
+    eventDescription?: string;
+    location?: string;
+    virtual?: boolean;
+    joinLink?: string;
+    joinLinkToPresenters?: number;
+    joinLinkToAttendees?: number;
+    companyId?: string;
+    managers?: string[];
+    speakers?: string[];
+}
+
+declare interface METMMiniEventCreateRequest extends METMMiniEventUpdateRequest{
+    eventId: string;
+    eventName: string;
+    eventDescription: string;
+    virtual: boolean;
+}
+
+declare interface METMMiniEvent extends METMMiniEventCreateRequest{
+    location?: string;
+    joinLink?: string;
+    joinLinkToPresenters?: number;
+    joinLinkToAttendees?: number;
+    companyId?: string;
+    managers?: string[];
+    speakers?: string[];

```

Comment:
I don't think any of these should be optional at this point, right?
```suggestion
    location: string;
    joinLink: string;
    joinLinkToPresenters: number;
    joinLinkToAttendees: number;
    companyId: string;
    managers: string[];
    speakers: string[];
```